src/modules/iotjs_module_process.c

```diff
@@ -146,12 +146,11 @@ JS_FUNCTION(Compile) {
 
   size_t size = 0;
   char* source = iotjs__file_read(filename, &size);
+  jerry_value_t jres = WrapEval(filename, strlen(filename), source, size);
```

Comment:
We should verify the source before `WrapEval`, what I meant is to throw the error when source is NULL.